demos/gallery/package.json

```diff
@@ -32,6 +32,8 @@
     "@hpcc-js/phosphor": "^2.5.1",
     "@hpcc-js/phosphor-shim": "^2.0.6",
     "@hpcc-js/timeline": "^2.1.2",
+    "@hpcc-js/preact-shim": "^2.0.5",
+    "@hpcc-js/timeline": "^2.1.0",
```

Comment:
I don't think this change was meant to be included?

Reply:
Yes, you're right. This is now a 1 line change.